src/driver/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["The Rust Project Developers"]
+name = "driver"
+version = "0.0.0"
```

Comment:
I think version field should be the same as rustc version, for all crates distributed with rustc.